moto/config/models.py

```diff
@@ -977,6 +980,7 @@ def list_aggregate_discovered_resources(
                 limit,
                 next_token,
                 resource_region=resource_region,
+                aggregator=self.config_aggregators.get(aggregator_name).__dict__,
```

Comment:
Is providing the aggregator needed following the other changes that were made?

Reply:
Hi There; sorry for the delay (just got back from holiday). The aggregator is needed - because it will need to know what regions to send back (either the `AllAwsRegions` flag or the list of `AwsRegions`).  I added a comment that tries to better explain it. I should be up-to-the-latest master and passing all the tests!

Reply:
Taking a look now